### ISSUE TITLE
Using the api without the query term

### ISSUE THREAD

**Issue by ghassanmas** [PorterK/GuardianJSClient#11]
So according to the Guardian documentation if the "q" parameter is not used the api will by default return the recent published article, a use case would be I just want to get the recent articles published by the guardian doesn't matter the category or the content of the article. 
Is that possible to accomplish using guardian-js?

**Comment by PorterK** [comment]
Yes

```js
api.custom.search()
```
will basically resolve to calling `http://content.guardianapis.com/search?api-key=<KEY>&?q=''`
which does give you the most recent stories in different categories

**Comment by PorterK** [comment]
This custom method isn't actually mentioned in the README at all, updating now. Thanks for bringing it up

**Comment by AJClassic2014** [comment]
Hi, PorterK

I installed API in create-react-app, I tested API as below in a js file
    `    api.content.search('football', {
      tag: 'sports'
    })
  .then(function(response){
    console.log(response);
  })
  .catch(function(err){
    console.log(err);
  });`

And I got nothing, it seems not working, and when I hover the mouse pointer on "const Guardian = require('guardian-js');"

Some notifications will occur as below:

"Could not find a declaration file for module 'guardian-js'. 'c:/Users/Yinfei/code_challenge/node_modules/guardian-js/src/index.js' implicitly has an 'any' type.
  Try `npm install @types/guardian-js` if it exists or add a new declaration (.d.ts) file containing `declare module 'guardian-js';` [7016]"

Did I use API wrong?
